dist/golang/protocol/fees.go

```diff
@@ -395,14 +395,13 @@ func EstimatedTransferResponse(requestTx *wire.MsgTx, dustLimit uint64, feeRate
 				fundingForTokens[i] += dustLimit
 			}
 
-		} else { // Output for BSV to track participant
-			for range asset.AssetSenders {
-				p2PKHOutputCount++
-				fundingForBSV[i] += dustLimit
-			}
-			if len(asset.AssetSenders) == 0 {
+		} else {
+			for _, sender := range asset.AssetSenders {
+				// For the amount that needs to be sent
+				fundingForBSV[masterContractIndex] += sender.Quantity
+				// For extra output that links to the sender
 				p2PKHOutputCount++
-				fundingForBSV[i] += dustLimit
+				fundingForBSV[masterContractIndex] += dustLimit
```

Comment:
This is adding dust on top of the actual bitcoin amount.

Reply:
Since the smart contract fix, dust is no longer added for BSV transfers.